app/views/declaration/items_summary.scala.html

```diff
@@ -18,49 +18,44 @@
 @import uk.gov.hmrc.play.views.html._
 @import models.declaration.governmentagencygoodsitem.GovernmentAgencyGoodsItem
 @import services.cache.ExportItem
+@import views.Title
 
 @this(main_template: views.html.main_template)
 
 
 @(mode: Mode, items: List[ExportItem], itemsErrors: Seq[FormError] = Seq.empty)(implicit request: Request[_], messages: Messages)
 
-@main_template(
-    title = messages("supplementary.items")
-) {
-    @components.back_link(routes.PreviousDocumentsController.displayPage(mode), mode)
+@defining(if(items.isEmpty) { messages("supplementary.itemsAdd.title") }
```

Comment:
I think use case for non, singular, plural use case would be quite ofter - maybe having simple apply method like
```
def apply(count: Int, none: String, singular: String, plural: String): Title = count match {
  case 0 => none
  case 1 => singular
  case _ => plural
}
```
As helper method for this case would simplify this code.

Extra: AFAIR java.text.format have support for conditional messages - maybe it would be worth to have quick look over topic.